src/crypto/qat/QatDrv.cmake

```diff
@@ -0,0 +1,23 @@
+##
+# Make file for QAT linux driver project
+##
+
+set(qat_driver_dir ${CMAKE_SOURCE_DIR}/src/crypto/qat/qat-driver)
+set(qatdrv_include -I${Boost_INCLUDE_DIRS})
+
+include(ExternalProject)
+ExternalProject_Add(QatDrv
+  DEPENDS Boost
+  URL "${qat_driver_dir}/qat1.7.upstream.l.1.0.3-42.tar.gz"
```

Comment:
should be using `https://01.org/sites/default/files/downloads/intelr-quickassist-technology/qat1.7.upstream.l.1.0.3-42.tar.gz` ,  i don't really think we should include a tar ball (which is a binary!) in our source directory.

Reply:
I am trying to confirm that the URL is a permanent one and nothing that will be dropped when the new package updates. As a backup i had it in the sources. Will switch to an URL as soon i as i get the confirmation